presto-hive/src/main/java/com/facebook/presto/hive/ParquetHiveRecordCursor.java

```diff
@@ -808,6 +805,26 @@ public ParquetListConverter(Type prestoType, String columnName, GroupType listTy
             }
         }
 
+        //copied over from Apache Hive
+        private boolean isElementType(parquet.schema.Type repeatedType, String parentName)
+        {
+            if (repeatedType.isPrimitive() ||
+                    (repeatedType.asGroupType().getFieldCount() != 1)) {
```

Comment:
In that case lets change this back to `> 1` since it would be invalid to have 0 or fewer groups